Description: Update Makefile to support qt5.
Author:  Scarlett Clark <sgclark@kde.org>
Last-Update: 2018-03-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -20,8 +20,8 @@
 BINDIR=$(PREFIX)/bin
 SHAREDIR=$(PREFIX)/share
 MANDIR=$(SHAREDIR)/man
-QMAKE_BINARY=qmake-qt4
-LRELEASE_BINARY=lrelease-qt4
+QMAKE_BINARY=qmake
+LRELEASE_BINARY=lrelease
 QMAKE_OPTS=
 
 LDFLAGS=
